android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -205,7 +205,7 @@ private synchronized void checkState(){
 		if(this.getState() == SETTING_UP
```

Comment:
Shouldn't we also check alow calling `checkState()` method on line 154 when level = LIMTED?

Reply:
Agreed! Added